go/registry/tests/tester.go

```diff
@@ -598,6 +625,9 @@ func (ent *TestEntity) NewTestNodes(nCompute int, nStorage int, runtimes []*node
 
 		// Add a registration with no P2P addresses.
 		invalid1 := *nod.Node
+		invalid1.P2P.ID = randomPK(rng)
```

Comment:
Aren't all these cases actually registration of a node with same `node.ID`, so it shouldn't be needed to set random IDs for each case?

Reply:
The P2P and Consensus IDs must differ in any case, so this is actually required.

Reply:
but aren't they already unique? (lines 614 and 617)

Reply:
Ah, good point :D  I've removed the unnecessary lines now.